evergreen/run-tests.sh

```diff
@@ -6,6 +6,8 @@ echo Testing with driver: "${DRIVER_VERSION}" "${TARGET_FRAMEWORK}"
 
 dotnet clean "./MongoDB.Analyzer.sln"
 
-dotnet build "./tests/MongoDB.Analyzer.Tests.Common.ClassLibrary" -f netstandard2.0 -c Debug
+dotnet build "./tests/MongoDB.Analyzer.Tests.Common.ClassLibrary" -f netstandard2.0 -c DRIVER_2_27_OR_LOWER
+
```

Comment:
Remove empty line (group together the ClassLibrary builds)

Reply:
Done